Ash.FrontEnd/src/js/global/components/Fido.jsx

```diff
@@ -112,6 +112,17 @@ class Fido extends React.Component {
         },
         {}
       );
+
+      // Call any new fetch configs with the callOnMount option set
+      const configs = this.getFullConfigs();
+      Object.keys(newFetchItems).forEach(key => {
+        const config = configs[key];
+
+        if (config.options.callOnMount) {
```

Comment:
Might want to make sure you list somewhere that callOnMount also calls on componentDidUpdate if it's a new config. Not intuitive from the name

Reply:
Renamed the option to `fetchImmediately` and updated its documenting comment